.github/workflows/release.yml

```diff
@@ -0,0 +1,73 @@
+name: Release
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+
+jobs:
+  release:
+    if: github.event.pull_request.merged == true && github.event.pull_request.head.ref == 'release'
+    runs-on: ubuntu-latest
+    container: cibuilds/github:0.13
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install libgcc
+        run: apk add libgcc
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: 1.51.0
+          target: wasm32-unknown-unknown
+          profile: minimal
+          override: true        
+      - name: Get version
+        id: get_version
+        run: |
+            VERSION=$(./devtools/get_version.sh)
+            echo ::set-output name=version::$VERSION
+        env:
+          HEAD_REF: ${{ github.event.pull_request.head.ref }}
+      - name: Create new tag
+        run: |
+          if [ -z "${{ steps.get_version.outputs.version }}" ]; then
+              git tag ${{ steps.get_version.outputs.version }}
+          fi
+      - name: Install Docker client
+        run: |
+          apk add docker-cli
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Prepare volume with source code
+        run: |
+          # create a dummy container which will hold a volume with config
+          docker create -v /code --name with_code alpine /bin/true
+          # copy a config file into this volume
+          docker cp ./Cargo.toml with_code:/code
+          docker cp ./contracts with_code:/code
+          docker cp ./packages with_code:/code
+      - name: Build development contracts
+        run: |
+          echo "Building all contracts under ./contracts"
+          docker run --volumes-from with_code cosmwasm/rust-optimizer:0.11.0 ./contracts/*/
```

Comment:
Does the volume mounting not working in the actions?

Reply:
I'm looking into ways to do this, but I need some time.

Reply:
Ok, let's enhance this later another issue.